pymodbus/client/tcp.py

```diff
@@ -97,7 +108,7 @@ def _create_protocol(self):
         protocol.factory = self
         return protocol
 
-    async def _connect(self):
+    async def _connect( self ):
```

Comment:
```suggestion
    async def _connect(self):
```
Please keep our standard